### PR TITLE
pagure fork token: purge it

### PR DESCRIPTION
**Pull request by TomasTomecek**
Fixes https://github.com/packit-service/packit/issues/495
Fixes packit-service/packit#395

### CODE REVIEW

**Comment by TomasTomecek** [comment]
Let's wait for:
* releasing packit 0.5.2
* deployment of pagure 5.7.7+ at src.fp.o

File: docs/docs/configuration/index.html [hunk 1]
```diff
@@ -1009,16 +1009,17 @@ <h3 id="values">Values</h3>
...
-<td><code>pagure_fork_token</code></td>
-<td>string</td>
-<td>a token so packit can create a pull request: <a href="https://src.fedoraproject.org/fork/YOU/rpms/PACKAGE/settings#apikeys-tab">https://src.fedoraproject.org/fork/YOU/rpms/PACKAGE/settings#apikeys-tab</a></td>
-</tr>
 </tbody>
 </table>
 
-<p>You can also specify the tokens as environment variables: <code>GITHUB_TOKEN</code>, <code>PAGURE_USER_TOKEN</code>, <code>PAGURE_FORK_TOKEN</code>.</p>
+<p>You can also specify the tokens as environment variables: <code>GITHUB_TOKEN</code>,
+<code>PAGURE_USER_TOKEN</code>.</p>
```
Comment (rpitonak) [inline docs/docs/configuration/index.html]: Is this still relevant after today arch meeting? You are right that it is still there, but we are going to deprecate it soon anyway.

File: content/docs/configuration.md [hunk 2]
```diff
@@ -233,9 +233,15 @@ The acceptable names are the same as for the package config:
  `fas_user`                   | string          | username in Fedora account system (to perform kinit if needed)
  `github_token`               | string          | Github API token: this is needed when packit interacts with Github API: https://github.com/settings/tokens (getting full read & write repo scope should be enough)
  `pagure_user_token`          | string          | Pagure token needed to access REST API, get it at: https://src.fedoraproject.org/settings#nav-api-tab
- `pagure_fork_token`          | string          | a token so packit can create a pull request: https://src.fedoraproject.org/fork/YOU/rpms/PACKAGE/settings#apikeys-tab
 
-You can also specify the tokens as environment variables: `GITHUB_TOKEN`, `PAGURE_USER_TOKEN`, `PAGURE_FORK_TOKEN`.
+You can also specify the tokens as environment variables: `GITHUB_TOKEN`,
```
Comment (rpitonak) [inline content/docs/configuration.md]: same here

**Comment by TomasTomecek** [comment]
@rpitonak good point, I added a note that it's deprecated now.

**Comment by TomasTomecek** [comment]
rebased and merging

**Pull request merged**